app/main.py

```diff
@@ -2,0 +1,16 @@
+from app.errors import (VaccineError, NotWearingMaskError)
+from app.cafe import Cafe
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
```

Comment:
Consider adding type hinting for the `cafe` parameter in the function definition. Type hinting can improve code readability and prevent potential bugs.